contracts/ReversibleICO.sol

```diff
@@ -617,7 +618,7 @@ contract ReversibleICO is IERC777Recipient {
         uint256 stageReturnedTokens
     ) {
 
-        ParticipantDetailsByStage storage totalsRecord = participantsByAddress[_address].byStage[_stageId];
+        ParticipantDetails storage totalsRecord = participantsByAddress[_address].byStage[_stageId];
```

Comment:
Is this still relevant if we now use `participantAggregatedStats[_participantAddress]`?
Also we have remove `participantsByAddress[_address].byStage`

Reply:
I think this is for returning only the token related stats and it is used mainly for testing, hence the naming in the return arguments, maybe @mickys  can help us on that. Are your referring about removing completely .byStage[_stageId]. I think if we really want it we can  think of a smart way to keep track of all stages, I can think about it more.

Reply:
Since the struct is public, this function it not needed in a strict way